tox.ini

```diff
@@ -95,12 +95,16 @@ setenv =
          #     .decode(getattr(sys.stdout, "encoding", "utf8"))
          # `TypeError: decode() argument 1 must be string, not None`
          PYTHONIOENCODING=utf_8
+
+	 # If no positional arguments are given, try to run the checks on the
+	 # entire codebase, including various pieces of supporting code.
+	 DEFAULT_FILES="src integration static misc setup.py"
 commands =
-         flake8 src integration static misc setup.py
-         python misc/coding_tools/check-umids.py src
-         python misc/coding_tools/check-debugging.py
-         python misc/coding_tools/find-trailing-spaces.py -r src static misc setup.py
-         python misc/coding_tools/check-miscaptures.py
+         flake8 {posargs:{env:DEFAULT_FILES}}
+         python misc/coding_tools/check-umids.py {posargs:{env:DEFAULT_FILES}}
+         python misc/coding_tools/check-debugging.py {posargs:{env:DEFAULT_FILES}}
+         python misc/coding_tools/find-trailing-spaces.py -r {posargs:{env:DEFAULT_FILES}}
```

Comment:
Quoting has the same effect here too: with quoted argument, these three scripts do nothing.